tests/css/test_mega_stylesheet.css

```diff
@@ -0,0 +1,227 @@
+/*
+ * The idea with this stylesheet is that it should contain as many valid
+ * ways of styling things as possible. So, as time goes on, feel free to add
+ * something to here. Be creative. Do the wildest things you can think of.
+ */
+
+* {}
+* * {}
+* * * {}
+* > * {}
+* > * > * {}
+
+/**********************************************************************/
+
+A {}
+A A {}
+A A A {}
+A > A {}
+A > A > A {}
```

Comment:
I see you have the pattern `X Y Z` and `X > Y > Z` but you skipped both `X Y > Z` and `X > Y Z`

Reply:
Yup, given how the rules seem to be constructed, either of those implies the other is fine. I just wanted to mostly kick off testing the one, two, many scenario.

We can add as many as we like from now on though. Adding stuff is encouraged!